src/MainWindow.hpp

```diff
@@ -47,7 +48,7 @@ class MainWindow : public QMainWindow {
     MainWindow();
 
     void closeEvent(QCloseEvent * event);
-    void preload(const std::vector<QString> & o) {
+    void preload(QStringList & o) {
```

Comment:
This is not an *out* parameter. Either `const &` or by value.

Reply:
Unintentionally changed that, sorry. Fixed.